src/stan_files/functions/jm_functions.stan

```diff
@@ -28,9 +28,9 @@
   * @param dist Integer specifying the type of prior distribution
   * @param scale Real specifying the scale for the prior distribution
   * @param df Real specifying the df for the prior distribution
-  * @return nothing
+  * @return Nothing
```

Comment:
that's not nothing

Reply:
Resolved in `jm_functions.stan` and `mvmer_functions.stan`. Now all say `@return lp__`.